LICENSE-NOTICE

```diff
@@ -0,0 +1,5 @@
+CHEF BINARY DISTRIBUTION AGREEMENT
+
+Use of this distribution of Habitat is subject to the terms and conditions of the Chef Binary Distribution Agreement. You can find the latest copy of the agreement here:
```

Comment:
Can we change this line and the top one to "Chef End User License Agreement" ?

Reply:
Yes sir